src/extension.ts

```diff
@@ -142,9 +142,10 @@ export async function activate(context: vscode.ExtensionContext) {
     const telemetryCallback = (...args: any[]) => {
```

Comment:
Lets change this  `(...args: any[]) => {` to `(...args: any[]): any => {` because by default it assumes return is `void`